docs/pages/components/inline-help.md

```diff
@@ -17,25 +17,33 @@ Inline help is used to display help text in a popover, often inline with headers
 ## Default Position
 The default positioning of inline help component is bottom right.
 {% capture inline-help %}
-
-<span class="fd-inline-help">
-    <span class="fd-inline-help__content fd-inline-help__content--bottom-right">
-        Lorem ipsum dolor sit amet, consectetur adipiscing.
-    </span>
-</span>
+<div class="fd-popover">
```

Comment:
I think this PR should be `feat:` as it brings changes in the markup

Reply:
Done